### ISSUE TITLE
Widget

### ISSUE THREAD

**Issue by mvarnagiris** [mvarnagiris/financius#23]

**Comment by thouseef** [comment]
I suggest lets start with a simple widget. My view is to have a simple view as follows:

Widget Layout::: [ icon |   Expense (Month) : 1250$ |  + ]

I'll do it in my spare time and let you know.

**Comment by mvarnagiris** [comment]
Sounds good.

**Comment by ansariNo1** [comment]
I'm desperate for the widget, it's a huge deal for usability! Currently, I'm using Expense Manager, which has a tidy little widget which allows me to enter a new transaction with a single click, which what the whole app is about. I can't leave this behind and move to an app that requires me to perform more clicks to carry out the basic functionality of the app.

**Comment by ghost** [comment]
I would love to see a widget there you can see the current balance on selected accounts

**Comment by xmas2** [comment]
similar widget to Expense Manager would be great (only to show current balance, not spent amount)

**Comment by pongthaic** [comment]
A large widget that let you enter new transaction amount and perhaps category would help.

**Comment by firnasz** [comment]
I would like to have 4 x 1 widget to see today's summary $ (+income-expenses) and a + button to create a transaction. Similar to Spendee widget.

**Comment by xenogew** [comment]
I would like widget that can make me to know my current balance and easy to add transactions.
And as firnasz said, the "Spendee" widget is the most similar to my idea.

**Comment by blakadder** [comment]
I agree, a simple widget like the one Expense manager has would suffice and make the new transaction entry much faster.

**Comment by lef-est** [comment]
I so want a widget too! This is why I left financius to monefy!

**Comment by lef-est** [comment]
And I love monefy's widget :)

**Comment by AFirooz** [comment]
Yes, please.

**Comment by shashwatblack** [comment]
A widget would be super useful. Just enter the amount and hit add.
Categories, tags etc could be added as well; possibly with transaction templates.
